docs/public/static/talks.ts

```diff
@@ -90,6 +90,20 @@ export const TALKS = [
 ] as Talk[];
 
 export const PODCASTS = [
+  {
+    title: 'What to do without App Center',
+    event: 'React Native Radio #315',
+    description: 'Quinlan Jung',
+    videoId: 'KBYxOy0g-rk',
+    link: 'https://reactnativeradio.com/episodes/rnr-315-what-to-do-without-app-center',
```

Comment:
```suggestion
    link: 'https://reactnativeradio.com/episodes/rnr-315-what-to-do-without-app-center',
    thumbnail: '...'
```

For RNR so far we were linking to podcast site, adding `videoId` changes those links to YT.

Reply:
Fetching thumbnails from their podcast site isn't working, so that's why I went with `videoId`. I guess I should replace these links with YT videos from their channel.

Reply:
FYI, we're using `videoId` + podcast link pattern for other podcasts as well.

Reply:
Updated back to use thumbnails and podcast links after downloading them, thanks for the tip 👍